### PR TITLE
[SKIP CI] add test for cli

### PR DESCRIPTION
**Pull request by jiqiang90**

### CODE REVIEW

File: packages/cli/src/controller/codegen-controller.spec.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    writeFile: jest.fn(),
+    access: jest.fn(),
+  };
+  fsMock.readFileSync = jest.fn();
+  return fsMock;
+});
+
+jest.mock('ejs', () => {
+  const ejsMock = jest.createMockFromModule('ejs') as any;
+  ejsMock.renderFile = jest.fn();
```
Comment (ianhe8x) [inline packages/cli/src/controller/codegen-controller.spec.ts:19]: This line seems not necessary, it has been done by `createMockFromModule`

File: packages/cli/src/controller/codegen-controller.ts [hunk 2]
```diff
@@ -8,33 +8,37 @@ import ejs from 'ejs';
 import {isNonNullType, getNullableType} from 'graphql';
 import {GraphQLFieldMap, GraphQLOutputType} from 'graphql/type/definition';
 import {transformTypes} from './types-mapping';
+
 const templatePath: string = path.resolve(__dirname, '../template/model.ts.ejs');
-const typesPath = `${process.cwd()}/src/types/models`;
 
 // 4. Save the rendered schema
-function makeSchema(className: string, data: string): void {
+export async function makeSchema(className: string, data: string): Promise<void> {
```
Comment (ianhe8x) [inline packages/cli/src/controller/codegen-controller.ts:15]: this abstraction doesn't make sense to me, it is simply wrap of fs.write and once be called once

**Review by ianhe8x — CHANGES_REQUESTED** [review]

File: packages/cli/src/controller/codegen-controller.spec.ts (hunk 1 above)
Comment (jiqiang90) [reply packages/cli/src/controller/codegen-controller.spec.ts]: removed

File: packages/cli/src/controller/codegen-controller.ts (hunk 2 above)
Comment (jiqiang90) [reply packages/cli/src/controller/codegen-controller.ts]: merged into renderTemplate

**Pull request closed without merging**